homeassistant/components/devolo_home_network/config_flow.py

```diff
@@ -100,9 +102,49 @@ async def async_step_zeroconf_confirm(
         if user_input is not None:
             data = {
                 CONF_IP_ADDRESS: self.context[CONF_HOST],
+                CONF_PASSWORD: "",
```

Comment:
Why do we store an empty password here but not in the user step?

Reply:
I added an empty password to the user step, too. It's in line 73.